UPGRADE.md

```diff
@@ -1,3 +1,8 @@
+### UPGRADE FROM 1.5.1 to 1.6
```

Comment:
It would be helpful to include information about PostgresSQL support and reference the `Sylius 1.13` upgrade note for guidance on performing migrations.

Reply:
I've added info to the UPGRADE.md